.github/workflows/helm-release.yaml

```diff
@@ -38,8 +34,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
-        with:
-          fetch-depth: 0
```

Comment:
We don't need this anymore? This config can be tested only when we release a new Helm version :(

Reply:
Nope we run git unshallow instead.